src/app/components/shell/notification-popover/notification-popover.ts

```diff
@@ -64,6 +64,8 @@ export default class AppShellNotificationPopover extends Vue {
 				slice: 15,
 				shouldScroll: false,
 				shouldShowUserCards: false,
+				// Some default value before the real feed data gets populated on show.
+				itemsPerPage: 15,
```

Comment:
why do we need this default?
I mean, what happens when the feed attempts to load with an unspecified amount of items per page?

Reply:
I built a check that this parameter is definitely set into the feed view, to make sure we don't fuck it up and forget.
This is the only place in the code where the feed gets bootstrapped without a feed payload, so in this specific instance we just set a count here.
It does get populated later, as the comment says, so it doesn't really matter.
The `15` is never actually used because when the feed gets populated, it also receives the perPage info from backend at the same time.